### PR TITLE
feat(pie-monorepo): add an empty nextjs example app

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Describe your changes (can list changeset entries if preferable)
[Added] - empty NextJS example application

### CODE REVIEW

File: apps/examples/wc-next13/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```
Comment (fernandofranca) [inline apps/examples/wc-next13/.eslintrc.json:2]: I believe this file is not necessary.

File: apps/examples/wc-next13/styles/globals.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
+:root {
```
Comment (fernandofranca) [inline apps/examples/wc-next13/styles/globals.css:1]: For demo/testing webcomponents context I believe we don't get any benefit by commiting this stylesheet.

File: apps/examples/wc-next13/styles/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,278 @@
+.main {
```
Comment (fernandofranca) [inline apps/examples/wc-next13/styles/Home.module.css:1]: I think this file can also be removed.

**Review by fernandofranca — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

File: apps/examples/wc-next13/next.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
```
Comment (raoufswe) [inline apps/examples/wc-next13/next.config.js:3]: I think we shouldn't use strict mode and stay consistent with the creat-react-app example as strict mode 

here are more details https://github.com/justeattakeaway/pie/pull/309#discussion_r1136929579

**Review by fernandofranca — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]
🔥

**Review by raoufswe — APPROVED** [review]

**Pull request merged**